tutorial/chapter1.md

```diff
@@ -0,0 +1,96 @@
+<p align="right">
+  <img width="25%" height="25%"src="../images/klepsydra_logo.jpg">
+</p>
+
+# Introduction
```

Comment:
Similar to kpsr-tutorial, add the technical requirements at the start.

Reply:
Done